app/src/main/java/org/p2p/wallet/bridge/model/BridgeBundleFee.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.wallet.bridge.model
+
+import java.math.BigDecimal
+
+data class BridgeBundleFee(
```

Comment:
BridgeAmount.kt can be used here instead of BridgeBundleFee to make common approach

Reply:
it's model from server, I don't have tokenSymbol here, let's discuss about it tomorrow